plugins/firebase/firebase.go

```diff
@@ -0,0 +1,25 @@
+package firebase
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func firebaseCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Firebase CLI",
+		Runs:    []string{"firebase"},
+		DocsURL: sdk.URL("https://firebase.google.com/docs/cli"),
+		NeedsAuth: needsauth.IfAll(
```

Comment:
Let's skip auth if the command contains the arg `--token` [because it takes precedence over envvars](https://firebase.google.com/docs/cli#cli-ci-systems):

<img width="888" alt="image" src="https://github.com/1Password/shell-plugins/assets/18581859/e3102f17-e320-4c1c-bb12-53c02a713331">